jwst/refpix/reference_pixels.py

```diff
@@ -269,6 +277,12 @@ def get_pixeldq(self):
             pixeldq[4:fullrows - 4, 0:4] = refpixdq_dontuse
             pixeldq[4:fullrows - 4, fullcols - 4:fullcols] = refpixdq_dontuse
             pixeldq[self.rowstart:self.rowstop, self.colstart:self.colstop] = self.input_model.pixeldq.copy()
+            if self.subarray in NRS_edgeless_subarrays:
+                # Log assignment as rows (in DMS plane) despite assigning columns (in detector plane)
+                log.info(f"Subarray {self.subarray} has no reference pixels: "
+                         f"assigning top and bottom four rows as reference pixels.")
+                pixeldq[self.rowstart:self.rowstop, self.colstart:self.colstart+4] = dqflags.pixel['REFERENCE_PIXEL']
```

Comment:
Is it OK to just assign them the REFERENCE_PIXEL value or should it be OR'd with any existing flags already set for those pixels in the pixeldq array? For example you don't want to wipe out any existing DO_NOT_USE flags, because it looks like there's a check for that later on.

Reply:
Good call; this change doesn't appear to significantly alter the output for my test data, but it should propagate the flags from dq_init if present.